samples/instance-config-get-operations.js

```diff
@@ -0,0 +1,72 @@
+/**
```

Comment:
Should file also be name instance-config-list-operations.js

Reply:
Have named it in similar lines of samples/backups-get-database-operations.js. Also the API we have exposed is spanner.getInstanceConfigOperations, so have named the file appropriately.

Reply:
We can change the method name, but not the documentation tag. Documentation tag is common across different clients, and should not be changed once decided.

Same thing is being done in `samples/backups-get-database-operations.js`.

Reply:
Incorporated the change.